trio/_highlevel_open_tcp_stream.py

```diff
@@ -364,7 +368,7 @@ async def attempt_connect(socket_args, sockaddr, attempt_failed):
             msg = "all attempts to connect to {} failed".format(
                 format_host_port(host, port)
             )
-            raise OSError(msg) from trio.MultiError(oserrors)
+            raise OSError(msg) from MultiError(oserrors)
```

Comment:
I think this one can become ExceptionGroup too, for the same reason

Reply:
Done.